_posts/2017/2017-08-08-why-you-should-spend-time-on-your-translation-keys.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Why you should spend time on your translation keys
+date: "2017-08-08 09:34:02"
+image: blog-header/translation-keys.jpg
+categories: tech-stories
+authors: ["Christina"]
+---
+Translation keys are not important at all?
+Spending time on them is a waste of time?
+Let me convince you otherwise.
+
+We just finished the revision of our translation keys because we noticed the value of a good key structure.
+In the following, I will tell you why we changed the old system and what have been the most important aspects during this task.
+
+## Structure
+
+We already had functional keys before.
+Everything worked out.
+But nobody really paid attention to an overall structure or any consistency.
+For our new technology platform, we will introduce a new localization process.
+Therefore, we will work with a new tool.
+A special feature of this tool is an In-Context Editor.
+This way, translations and wording can be handled directly in the software and translators as well as UI-Writers are provided with as much context as possible.
+They know exactly where the keys are located and which kind of wording is needed.
+Thinking of the save button, you will understand why context is that important.
```

Comment:
Add headline before this sentence "The importance of context" or something like that. Otherwise, the following paragraph comes so "out of nowhere" ;-)